relevanceai/operations/operations.py

```diff
@@ -49,21 +47,36 @@ def auto_cluster(
             credentials=self.credentials,
             model=model,
             alias=alias,
-            **kwargs,
-        )
-        return ops(
             dataset_id=self.dataset_id,
             vector_fields=vector_fields,
+            **kwargs,
         )
+        return ops(dataset_id=self.dataset_id, vector_fields=vector_fields)
 
     def dr(
         self,
-        model,
-        n_components,
-        alias,
-        vector_fields,
+        model: Any,
```

Comment:
`model: Union[str, Any]` Users should be able to pass a string

Reply:
@jtwinrelevanceai Any includes `str`